src/Database/Database.php

```diff
@@ -65,6 +65,15 @@ class Database
     // Cache
     const TTL = 60 * 60 * 24; // 24 hours
 
+    // Events
+    const EVENT_CREATE_DATABASE = 'create_database';
```

Comment:
```suggestion
    const EVENT_DATABASE_CREATE = 'database_create';
```

Makes more sense to have the namespace before the action